config/options

```diff
@@ -87,7 +87,9 @@ VERBOSE="${VERBOSE:-yes}"
 CCACHE_CACHE_SIZE="10G"
 
 # read options from $HOME if available
-if [ -f "${HOME}/.libreelec/options" ]; then
+if [ -f "${ROOT}/.libreelec/options" ]; then
```

Comment:
Two separate if blocks, $HOME first then $ROOT? This would let you store personal global options in HOME, and then tree specific in ROOT.

Reply:
I wondered about that too, but figured it might become too complex if you then had to disable an option in `$ROOT/.libreelec/option` that has been set by by `$HOME`, or you add something in `$HOME` that means a previously working branch with `$ROOT/.libreelec/options` no longer works (or worse, works subtly and unintentionally differently than it did before).

Simplest solution would be for the user to manage all `.libreelec/options` in either `$ROOT` or `$HOME` but not both, as implemented by this PR.

Reply:
I'd also vote for 2 if blocks - HOME first, then ROOT.

This follows the other option sourcing logic in config/options, allowing overriding previously set options (distro->project->device->HOME previously).

It also makes a lot of sense to put common options (like BUILDER_NAME) in HOME so all of these don't need to be replicated in ROOT options.

The concept of ROOT options seems very beneficial to me (already had the need for that before, so I could set different options for LE 9.2 and master builds), but I'd like to be able to set stuff common for all builds, like BUILDER_NAME, CCACHE_CACHE_SIZE, SYSTEM_SIZE, ADDON_OVERWRITE, SOURCES_DIR, CONFIG_SHELL, DISABLE_COLORS in a single place - $HOME/.libreelec/options